Files/Interacts/Interaction.cs

```diff
@@ -1048,7 +1048,8 @@ public async void PasteItem_ClickAsync(object sender, RoutedEventArgs e)
             await PasteItems(packageView, destinationPath, packageView.RequestedOperation);
         }
 
-        public async Task PasteItems(DataPackageView packageView, string destinationPath, DataPackageOperation acceptedOperation)
+        public async Task PasteItems(DataPackageView packageView, string destinationPath, DataPackageOperation acceptedOperation,
+            bool copyDuplicate = true)
```

Comment:
Remove `copyDuplicate`

Reply:
I created tit to address #1546, but since this isn't the scope of the pr I will remove it for now and address in a next pr.

Reply:
Made requested changes.